kolibri/plugins/facility/assets/src/views/DataPage/index.vue

```diff
@@ -22,6 +22,8 @@
               class="download-button"
               @click="downloadSessionLog"
             />
+            <span v-if="noSessionLogs"> {{ $tr('noLogsYet') }} </span>
```

Comment:
```suggestion
            <span v-if="noSessionLogs">{{ $tr('noLogsYet') }}</span>
```

For code style: remove the extra space inside the tags

Reply:
Agreed, and not just for style: sometimes extraneous whitespace causes confusing rendering bugs.